Modules/StudyProgramme/classes/tables/class.ilIndividualPlanGUI.php

```diff
@@ -7,6 +7,8 @@
  *
  * @author Stefan Hecken 	<stefan.hecken@concepts-and-training.de>
  */
+require_once("Services/GEV/Utils/classes/class.gevUserUtils.php");
```

Comment:
Please do not rely on customer specific utils here; you want a user's full name, which is pretty much standard in ILIAS.